optuna/logging.py

```diff
@@ -175,6 +175,33 @@ def disable_propagation() -> None:
     """Disable propagation of the library log outputs.
 
     Note that log propagation is disabled by default.
```

Comment:
How about modifying it to `Note that log propagation is disabled by default. You only need to use `disable_propagation` to stop log propagation when you use `enable_propagation`.

Reply:
Thank you for reviewing, seems more user friendly and readable. I'll change that.